db/schema.rb

```diff
@@ -10,50 +10,49 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[6.1].define(version: 2021_10_19_095600) do
```

Comment:
There seem to be quite a few unrelated changes to the `schema.rb` file that appear related to the Rails 7 upgrade.  Maybe we missed a step during the upgrades?  In any case, this probably ought to be in another commit (or ideally PR) to keep those changes separate.

Reply:
Good point, thank you @brucebolt